sap/adt/objects.py

```diff
@@ -233,7 +233,9 @@ def get_uri_for_type(self, mimetype):
         """
 
         try:
-            return '/' + self._typeuris[mimetype]
+            mimetype_uri = self._typeuris[mimetype]
+
+            return '/' + mimetype_uri if mimetype_uri != '' else ''
```

Comment:
I would put this into a solo commit and explain it carefully.

Reply:
Done in commit 760ad1e